### PR TITLE
CORE-14203: Fix Session Event Wrapping

### PR DESCRIPTION
**Pull request by jujoramos**
Ensure that all messages sent to 'p2p.out' topic are wrapped as
instaces of 'AppMessage' to prevent deserialization issues.

### CODE REVIEW

**Review by thiagoviana — DISMISSED** [review]

**Review by JamesHR3 — DISMISSED** [review]
@LWogan @emilybowe - think this is the fix we were talking about yesterday?

LGTM

**Review by emilybowe — DISMISSED** [review]
LGTM

File: libs/flows/flow-mapper-impl/src/test/kotlin/net/corda/flow/mapper/impl/executor/SessionInitExecutorTest.kt [hunk 1]
```diff
@@ -142,4 +142,4 @@ class SessionInitExecutorTest {
 
         assertThat(resultOutbound.outputEvents).isEmpty()
```
Comment (LWogan) [inline libs/flows/flow-mapper-impl/src/test/kotlin/net/corda/flow/mapper/impl/executor/SessionInitExecutorTest.kt:147]: i think we should assert that the class type of the output event is AppMessage

Comment (jujoramos) [reply libs/flows/flow-mapper-impl/src/test/kotlin/net/corda/flow/mapper/impl/executor/SessionInitExecutorTest.kt:147]: Done 👍

**Review by LWogan — APPROVED** [review]
LGTM

**Review by rick-r3 — APPROVED** [review]

**Pull request merged**